js/layout-javascript/news-feed-code.js

```diff
@@ -833,6 +868,29 @@ DynamicList.prototype.removeListItemHTML = function (options) {
   this.$container.find('.news-feed-list-item[data-entry-id="' + id + '"]').remove();
 }
 
+DynamicList.prototype.resetSortIcons = function() {
+  var $allListItems = $('#sort-list-news').find('li');
+
+  $allListItems.each(function() {
+    var $listitem = $(this);
+    var $listSortOrder = $listitem.data('sortOrder');
+    var $listIcon = $listitem.find('i');
+
+    switch ($listSortOrder) {
+      case 'asc':
+        $listIcon.removeClass('fa-sort-asc').addClass('fa-sort');
```

Comment:
No `switch` necessary.

```suggestion
        $listIcon.removeClass('fa-sort-' + listSortOrder).addClass('fa-sort');
```